networking/firewall/000infralib/runtest.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# vim: dict=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /kernel/networking/firewall/000infralib
+#   Description: firewall/000infralib
+#   Author: Shuang Li <shuali@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2016 Red Hat, Inc. All rights reserved.
+#
+#   This copyrighted material is made available to anyone wishing
+#   to use, modify, copy, or redistribute it subject to the terms
+#   and conditions of the GNU General Public License version 2.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE. See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Global parameters
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#
+# Select tool to manage package, which could be "yum" or "dnf"
+# Note global variable YUM will be used later
+#
+FILE=$(readlink -f ${BASH_SOURCE})
+NETWORKING_ROOT="${FILE%/networking/*}/networking"
+source $NETWORKING_ROOT/common/include.sh
+yum=$(select_yum_tool)
+if (( $? != 0 )); then
+	echo "FATAL: fail to get package tool" >&2
+	exit 1
```

Comment:
Could this abort? We don't want to report a kernel problem to developers because of the fail. (Not that it should ever hit this condition)

Reply:
Hi @veruu, do we need to `exit 0` instead?

Reply:
Hi @idorax , it should report infrastructure failure like for example here: https://github.com/CKI-project/tests-beaker/blob/master/distribution/kpkginstall/runtest.sh#L297

Reply:
Hi @veruu, I have updated `networking/firewall/000infralib/runtest.sh` according to your suggestion, please take a look again, thanks!